tests/installation/add_maintenance_repos.pm

```diff
@@ -0,0 +1,31 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Add new add-ons using maintenance test repo URLs.
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi 'get_var';
+
+sub run {
+    set_var('MAINT_TEST_REPO', get_var('INCIDENT_REPO')) if get_var('INCIDENT_REPO');
+    my @repos = split(/,/, get_var('MAINT_TEST_REPO'));
+
+    $testapi::distri->get_add_on_product()->confirm_like_additional_add_on();
+
+    while (defined(my $maintrepo = shift @repos)) {
+        $testapi::distri->get_add_on_product()->accept_current_media_type_selection();
+        $testapi::distri->get_repository_url()->add_repo({url => $maintrepo});
+        if (@repos) {
```

Comment:
i cant see the reason for this /if/ condition. The results would be identical with or without. right?

Reply:
I believe it's a remnant from an earlier version where I pressed next as a last step in the loop, instead of outside. I removed it and running a VR to make sure all is ok. Thanks!

Reply:
@b10n1k my bad - this is actually needed. We want to add another repo only if there's another repo to add. If `repos` is depleted, we want to move to the next page and not add another one.

Reply:
maybe add this small detail as a comment? it looks a bit weird in first glance but its up to you and your team. i am gonna resolve the conversation for now